src/workflow_docs/fetch_docs_sources.sh

```diff
@@ -16,18 +16,26 @@ git clone --no-checkout --depth=1 --single-branch --branch=master --filter=tree:
 git clone --no-checkout --depth=1 --single-branch --branch=master --filter=tree:0 https://github.com/microbiomedata/mg_annotation /tmp/clones/mg_annotation         && cd /tmp/clones/mg_annotation     && git sparse-checkout set --no-cone /docs && git checkout
 git clone --no-checkout --depth=1 --single-branch --branch=master --filter=tree:0 https://github.com/microbiomedata/metaMAGs /tmp/clones/metaMAGs                   && cd /tmp/clones/metaMAGs          && git sparse-checkout set --no-cone /docs && git checkout
 git clone --no-checkout --depth=1 --single-branch --branch=main   --filter=tree:0 https://github.com/microbiomedata/metaT /tmp/clones/metaT                         && cd /tmp/clones/metaT             && git sparse-checkout set --no-cone /docs && git checkout
+git clone --no-checkout --depth=1 --single-branch --branch=main   --filter=tree:0 https://github.com/microbiomedata/metaT_ReadsQC /tmp/clones/metaT_ReadsQC
+git clone --no-checkout --depth=1 --single-branch --branch=main   --filter=tree:0 https://github.com/microbiomedata/metaT_ReadsQC /tmp/clones/metaT_Assembly
+git clone --no-checkout --depth=1 --single-branch --branch=main   --filter=tree:0 https://github.com/microbiomedata/metaT_ReadsQC /tmp/clones/metaT_ReadCounts
```

Comment:
This is cloning a Git repo three times, into three different directories. Since these lines don't have the "checkout" part, the only thing that will exist in those directories is a `.git` directory.

I don't know what you were trying to accomplish with these lines. If you send me that info, I'll provide some feedback (or make the relevant changes to this file).

Reply:
those had the incorrect source full path, they are updated now.